linux/mprisService.js

```diff
@@ -1,4 +1,5 @@
-const { ipcMain } = require('electron');
+const { ipcMain } = require('electron'); // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
Instead of disabling this inline, lets add it to the ignore part of the eslintrc